skiko/src/commonMain/kotlin/org/jetbrains/skia/Paint.kt

```diff
@@ -674,21 +629,6 @@ private external fun _nGetStrokeJoin(ptr: NativePointer): Int
 @ExternalSymbolName("org_jetbrains_skia_Paint__1nSetStrokeJoin")
 private external fun _nSetStrokeJoin(ptr: NativePointer, value: Int)
 
-@ExternalSymbolName("org_jetbrains_skia_Paint__1nGetFillPath")
```

Comment:
Were these 2 functions removed in Skia? 
Skiko never stated it's stable, so I'm just curious if it can ever become stable if skia would have such breaking changes.

Reply:
For Skia it's pretty common to introduce breaking changes in the API between major releases. https://github.com/google/skia/blob/aab9fb4100da797d25fe340e9a2fcb2ae30fc2e1/RELEASE_NOTES.txt#L10
I think we could do the same for Skiko.

Reply:
We can garanty stability `org.jetbrains.skiko` in the future, but not `org.jetbrain.skia`